test/it/uk/gov/hmrc/exports/repositories/NotificationRepositorySpec.scala

```diff
@@ -165,10 +175,87 @@ class NotificationRepositorySpec
         repo.save(notification).futureValue
         repo.save(notification_2).futureValue
         repo.save(notification_3).futureValue
+        repo.save(notificationUnparsable).futureValue
 
         repo.findNotificationsByActionIds(Seq.empty).futureValue must equal(Seq.empty)
       }
     }
   }
 
+  "Notification Repository on findUnparsedNotifications" when {
+
+    "there are no unparsed Notification" should {
+      "return empty list" in {
+        repo.save(notification).futureValue
+
+        repo.findUnparsedNotifications().futureValue must equal(Seq.empty)
+      }
+    }
+
+    "there is one unparsed Notification" should {
+      "return those Notifications" in {
+        repo.save(notification).futureValue
+        repo.save(notificationUnparsable).futureValue
+
+        val foundNotifications = repo.findUnparsedNotifications().futureValue
+
+        foundNotifications.length must equal(1)
+        foundNotifications must contain(notificationUnparsable)
+      }
+    }
+
+    "there are many unparsed Notification" should {
+      "return those Notifications" in {
+        val anotherUnparsableNotification = notificationUnparsable.copy(actionId = "c1c09f1b-7c94-4e90-b754-7c5c71c55e44")
```

Comment:
Nit: In most our repositories there is an object `CommonTestData`. It contains examples on various identifiers, like DUCR, MUCR, MRN, ConversationId. You can use it so that you don't need to hard code them in tests.

Reply:
Ok, just wanted something that wasn't already used. I've modified it to use a pre-defined value from ExportsTestData now.